src/components/SightingDetail/SightingDetail.tsx

```diff
@@ -0,0 +1,38 @@
+import "./SightingDetail.scss"
+import React, { useEffect, useState } from 'react';
+import { fetchSightingById, WhaleSighting} from "../../clients/apiClient";
+import "./SightingDetail.scss";
+
+interface SightingDetailProps {
+    sightingId: number;
+}
+export function SightingDetail(props: SightingDetailProps): JSX.Element {
```

Comment:
Can we change the syntax here so when we use the id we don't have to call `props.sightingId` it can just be `sightingId` directly
```suggestion
export function SightingDetail({sightingId}: SightingDetailProps): JSX.Element {
```